SECURITY-INSIGHTS.yml

```diff
@@ -1,6 +1,6 @@
 header:
   schema-version: 1.0.0
-  expiration-date: '2024-12-17T01:00:00.000Z'
+  expiration-date: '2026-06-18T01:00:00.000Z'
```

Comment:
is this something that needs to be updated manually ? is there any guidance or documentation we can follow